### DISCUSSION TITLE
Existing compiler projects on GitHub

### DISCUSSION THREAD

**Discussion by LoopGlitch26** [hackclubiter/onEditor#9] [Resources]
https://github.com/jamiebuilds/the-super-tiny-compiler

https://github.com/llSourcell/Lets_Build_a_Compiler_LIVE

https://github.com/shawon100/Online-Compiler-Windows-Server

https://github.com/shawon100/Online-Compiler

https://github.com/droidlife/online-compiler-api

https://github.com/Jaagrav/CodeX

https://github.com/akashgp09/Online-Compiler-Web

https://github.com/zakariamaaraki/RemoteCodeCompiler

https://github.com/bhargavgolla/Online-Compiler

https://github.com/sonnylazuardi/node-compiler

https://github.com/hermanzdosilovic/online-compiler

...To be updated

**Comment by LoopGlitch26** [comment]
I think this is a better place to see all the important repos all at once:

https://github.com/topics/online-compiler

**Comment by saswatsamal** [comment]
[This](https://secure.judge0.com/extra) looks promising and helpful. (from Judge0)

**Comment by LoopGlitch26** [comment]
https://github.com/sahildua2305/hackIDE/tree/master

**Comment by LoopGlitch26** [comment]
https://github.com/GalaxyZpj/Judge0
